src/features/stackQueue/components/StackQueueVisualizer.jsx

```
@@ -141,61 +169,67 @@ const StackQueueVisualizer = () => {
             setCurrentStep("Stack has elements, proceeding with pop");
           }
           break;
-        case 4: // Pop operation
+        case 4:
           setCurrentLine(22);
           setCurrentStep("Removing top element");
           const poppedValue = elements[elements.length - 1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)